src/dmd/dtemplate.d

```diff
@@ -476,6 +476,13 @@ extern (C++) final class Tuple : RootObject
 {
     Objects objects;
 
+    extern (D) this(){}
+
+    extern (D) this(size_t numObjects)
```

Comment:
Yes of course. This is just a bit strange to start doing it when the practice is not generalized.
I had to amend anyway since the default ctor can be avoided using a default numObjects.